js/functions.js

```diff
@@ -1,3 +1,25 @@
+/*
+'8:00' - начало рабочего дня
+'17:30' - конец рабочего дня
+'14:00' - начало встречи
+90 - продолжительность встречи в минутах
+*/
+
+function isWithinWorkingHours(workdayStart, workdayEnd, meetingStart, meetingDuration) {
+  const reverseToMinutes = (time) => {
+    const [hours, minutes] = time.split(':');
+
+    return +hours * 60 + +minutes;
+  };
+
+  const workStart = reverseToMinutes(workdayStart);
+  const workEnd = reverseToMinutes(workdayEnd);
+  const meetStart = reverseToMinutes(meetingStart);
+  const meetEnd = meetStart + meetingDuration;
+
+  return meetEnd <= workEnd && workStart <= meetStart;
+}
```

Comment:
```suggestion
function isMeetingWithinWorkHours(workdayStart, workdayEnd, meetingStart, meetingDuration) {
  const timeToMinutes = (time) => {
    const [hours, minutes] = time.split(':').map(Number);
    return hours * 60 + minutes;
  };

  const workStart = timeToMinutes(workdayStart);
  const workEnd = timeToMinutes(workdayEnd);
  const meetStart = timeToMinutes(meetingStart);
  const meetEnd = meetStart + meetingDuration;

  return meetEnd <= workEnd && meetStart >= workStart;
}
```

Не big deal, но я тут сделал несколько улучшений:
1. Переименовал функцию, для большей ясности
2. Переименовал и чуть-чуть порефакторил вспомогательную функцию, теперь в ней меньше операторов и читать проще немного
3. Переделал return, чтобы было тоже чуть более читаемо